frontend/src/metabase-types/api/mocks/query.ts

```diff
@@ -5,9 +5,15 @@ import type {
   StructuredQuery,
 } from "metabase-types/api";
 
+// Corresponds to the Sample Database and sample Products table from the built-in preset:
+// https://github.com/metabase/metabase/blob/master/frontend/src/metabase-types/api/mocks/presets/sample_database.ts
+const DEFAULT_DB_ID = 1;
```

Comment:
We use `1` for all databases and tables created by these functions. I wouldn't imply here that it has something to do with out sample db

Reply:
Yeah, I think you're right. Updated 👍